requirements.txt

```diff
@@ -2,4 +2,4 @@ numpy
 scipy>=0.9
 # scikit-learn
 pandas
-
+pybedtools
```

Comment:
huh, I thought this would have solved the issue. 

@AaronWatters , do you happen to know why this line is not fixing the requirements problem? According to the log on travis CI for this commit, only the following packages were installed in the setup:

The following NEW packages will be INSTALLED:
    cairo:           1.12.18-6         
    coverage:        4.2-py27_0        
    cycler:          0.10.0-py27_0     
    fontconfig:      2.11.1-6          
    freetype:        2.5.5-1           
    glib:            2.43.0-1          
    harfbuzz:        0.9.39-1          
    libffi:          3.2.1-0           
    libgfortran:     3.0.0-1           
    libpng:          1.6.22-0          
    libxml2:         2.9.2-0           
    matplotlib:      1.5.1-np111py27_0 
    mkl:             11.3.3-0          
    nose:            1.3.7-py27_1      
    numpy:           1.11.1-py27_0     
    openssl:         1.0.2h-1          
    pandas:          0.18.1-np111py27_0
    pango:           1.39.0-1          
    pip:             8.1.2-py27_0      
    pixman:          0.32.6-0          
    pycairo:         1.10.0-py27_0     
    pyparsing:       2.1.4-py27_0      
    pyqt:            4.11.4-py27_4     
    python:          2.7.12-1          
    python-dateutil: 2.5.3-py27_0      
    pytz:            2016.6.1-py27_0   
    qt:              4.8.7-4           
    readline:        6.2-2             
    scikit-learn:    0.17.1-np111py27_2
    scipy:           0.18.0-np111py27_0
    setuptools:      25.1.6-py27_0     
    sip:             4.18-py27_0       
    six:             1.10.0-py27_0     
    sqlite:          3.13.0-0          
    tk:              8.5.18-0          
    wheel:           0.29.0-py27_0     
    zlib:            1.2.8-3